ts/Core/Chart/StockChart.ts

```diff
@@ -725,17 +725,20 @@ addEvent(Axis, 'afterDrawCrosshair', function (
         point[this.isXAxis ? 'x' : 'y'] :
         this.toValue(horiz ? e.chartX : e.chartY);
 
+    // Crosshair should be rendered within Axis range (#7219)
+    // Also, the point of currentPriceIndicator
+    // should be inside the plot area, #14879.
+    const isInside = point ? point.series.isPointInside(point) :
+        (value as any) > (min as any) || (value as any) < (max as any);
```

Comment:
You changed `<` and `>` operators to be opposite - be careful with that :)

Reply:
Indeed, I changed that to make it more clear and readable. 
I also changed the visibility values in the ternary operator in order to preserve the logic. 
I checked these changes once again and everything works as expected.

Reply:
Sorry for not explaining enough.
The old condition `!(A+B)` is not the same as `!A+!B`. I should be `!A*!B`.

Your condition always returns `true`. Let's take `value = 15`, `min = 0` and `max = 10`
`15 > 1 || 15 < 10` <==> `true || false` <==> `true` but, obviously, 15 is not between 0 and 10.

Also, when it comes to negation, remember that `!<` is `>=`, not `>`

Reply:
You are right! 
Thanks for pointing that out! ;)

Reply:
No problem.
Have you checked whether it should be `>` or `>=`?

Reply:
I didn't find a single test case in which `>` will fail.